### PR TITLE
Enable VMExport feature gate by default

### PR DESCRIPTION
**Pull request by akalenyu**
Allows exporting VMs outside of the cluster.

Signed-off-by: Alex Kalenyuk <akalenyu@redhat.com>

**Reviewer Checklist**

> Reviewers are supposed to review the PR for every aspect below one by one. To check an item means the PR is either "OK" or "Not Applicable" in terms of that item. All items are supposed to be checked before merging a PR. 

- [ ] PR Message
- [ ] Commit Messages
- [ ] How to test
- [ ] Unit Tests
- [ ] Functional Tests
- [ ] User Documentation
- [ ] Developer Documentation
- [ ] Upgrade Scenario
- [ ] Uninstallation Scenario
- [ ] Backward Compatibility
- [ ] Troubleshooting Friendly
  
**Release note**:

```release-note
Enable the VMExport feature gate for kubevirt
```

### CODE REVIEW

File: controllers/operands/kubevirt.go [hunk 1]
```diff
@@ -106,7 +106,6 @@ var (
 		kvHostDevicesGate,
 		kvDownwardMetricsGate,
 		kvNUMA,
-		kvLiveMigrationGate,
```
Comment (akalenyu) [inline controllers/operands/kubevirt.go]: Are we ok with this?

Comment (nunnatsa) [reply controllers/operands/kubevirt.go]: Yes. This is part of #2043

Comment (akalenyu) [reply controllers/operands/kubevirt.go]: weird that it shows up here as a diff

**Review by orenc1 — APPROVED** [review]
/approve

**Pull request merged**